run_one_experiment.py

```diff
@@ -79,6 +80,10 @@ def __str__(self):
         f'max coverage diff sample: {self.max_coverage_diff_sample}\n'
         f'max coverage diff report: {self.max_coverage_diff_report or "None"}')
 
+  def __post_init__(self, *args, **kwargs):  # pylint: disable=unused-argument
+    if not self.gained_textcov:
+      self.gained_textcov = textcov.Textcov()
```

Comment:
also can we use https://docs.python.org/3/library/dataclasses.html#dataclasses.field instead to initialise this instead?